common.go

```diff
@@ -102,6 +103,7 @@ func (terms dbTermList) crush() dbRecordList {
 			strings.Join(t.Rules, " "),
 			t.Score,
 			t.Glossary,
+			t.Sequence,
```

Comment:
I don't know off-hand if the sequence values for JMDict are 0-based or 1-based, but if they are 0-based then this should probably be set to `t.Sequence + 1`. The reason for this is so that a sequence value of `0` can become a sentinel for "no sequence defined".

Reply:
Seems like they start at 1000000, but I don't know how it's determined. In Yomichan, the sentinel value is currently set to `-1`.

Reply:
Using `-1` works as well, but in that case you'll have to make sure that all the other dictionaries that do not have the concept of `Sequence` have it initialized to that value.

Reply:
Yeah, thinking about it more `-1` is probably most explicit (since it's obviously an invalid sequence). You'll just have to make sure that the EPWING and ENAMDICT parsers explicitly set this value (since the default of `0` would be a valid value).